frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -60,15 +60,18 @@ export const MonitoringAlerts: React.FC<props> = ({ match, rules, filters, listS
     POLL_DELAY,
     namespace,
   );
-  const thanosRules = React.useMemo(
-    () => (!loading && !loadError ? getAlertsAndRules(response?.data).rules : []),
+  const thanosAlertsAndRules = React.useMemo(
+    () => (!loading && !loadError ? getAlertsAndRules(response?.data) : { rules: [], alerts: [] }),
     [response, loadError, loading],
   );
 
   React.useEffect(() => {
-    const sortThanosRules = _.sortBy(thanosRules, (rule) => alertingRuleStateOrder(rule));
+    const sortThanosRules = _.sortBy(thanosAlertsAndRules.rules, (rule) =>
+      alertingRuleStateOrder(rule),
+    );
```

Comment:
```suggestion
    const sortThanosRules = _.sortBy(thanosAlertsAndRules.rules, alertingRuleStateOrder);
```

Reply:
I will address it in https://github.com/openshift/console/pull/5943.